### ISSUE TITLE
Pilot Siteimprove Accessibility Tool in the CMS

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#15998]
## Description of Work

How might we pilot Siteimprove (an accessibility scanning tool) to validate its ability to provide in-CMS guidance for VAMC editors to create accessible and plain language content on VA.gov?

## Goal

Determine how best to provide VAMC editors (as a pilot group) with accessibility and content guidance at the point of creation. 

**Our Bet:** Leveling up the editorial content creation process with the Siteimprove tool will result in fewer errors and more reliable content on VA.gov and associated experiences. 

## Details

OCTO Lead- Erika Washburn
VA Stakeholders- VHA DM 
Is this project tracked to an upcoming event or congressional deadline? No

What is the user impact and scale? 
This pilot will validate the usefulness of Siteimprove as a tool within the Drupal CMS that will provide real-time feedback and scanning of content and associated media such as PDFs. The pilot will only be for the VAMC product and during a 30-day free trial period as a chance to validate its usefulness. If UX, accessibility, and technical testing passes, this tool could be validated as an option for full-scale roll out.

**Comment by EWashb** [comment]
Waiting on support from Siteimprove user support to provide a prepublish token for Drupal integration

**Comment by BerniXiongA6** [comment]
cc: @maortiz-27-80

**Comment by EWashb** [comment]
We have identified an issue with our decoupled system and intended integration with the SiteImprove API. Likely we will not be able to pursue this until either content-build is modified or we are entirely on Next.js. I'll keep this as on hold until I can get more information about the technical feasibility. 

however, we were able to implement deeplinking for a couple of sites to pilot. This deeplinking approach allows users to access the issues reported via Siteimprove directly to the source of the issue within the CMS
